ansible/playbooks/roles/encoder-network/tasks/main.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Ensure nftables is installed
+  apt:
+    package: nftables
+    state: latest
+
+- name: Create nftables config file
+  copy:
+    src: nftables.conf
+    dest: /etc/nftables.conf
+
+- name: Activate nftables config
+  service:
+    name: nftables
+    state: restarted
```

Comment:
This is not idempotent, move this to `handlers/main.yml` and use a `notify` on the config file.